docker_images/peft/tests/test_api.py

```diff
@@ -0,0 +1,61 @@
+import os
+from typing import Dict
+from unittest import TestCase, skipIf
+
+from app.main import ALLOWED_TASKS, get_pipeline
+
+
+# Must contain at least one example of each implemented pipeline
+# Tests do not check the actual values of the model output, so small dummy
+# models are recommended for faster tests.
+TESTABLE_MODELS: Dict[str, str] = {
```

Comment:
You also need to have a model here, otherwise `docker_images/peft/tests/test_api_text_generation.py` will never run